CRM/Membershipextrasimporterapi/EntityImporter/Contribution.php

```diff
@@ -279,9 +281,28 @@ private function createEntityFinancialTransactionRecord($contributionId, $financ
     $sqlParams = [
       1 => [$mappedContributionParams['total_amount'], 'String'],
     ];
-    $sqlQuery = "INSERT INTO `civicrm_entity_financial_trxn` (`entity_table`, `entity_id` , `financial_trxn_id`, `amount`) 
+    $sqlQuery = "INSERT INTO `civicrm_entity_financial_trxn` (`entity_table`, `entity_id` , `financial_trxn_id`, `amount`)
                 VALUES ('civicrm_contribution', {$contributionId}, {$financialTransactionId}, %1)";
     SQLQueryRunner::executeQuery($sqlQuery, $sqlParams);
   }
 
+  /**
+   * Sets the contribution owner organisation,
+   * which is only needed if Multicompany accounting
+   * is used (Multicompany accounting extension is enabled).
+   *
+   * @param $contributionId
+   *
+   * @return void
+   */
+  private function setOwnerOrganisationIfExist($contributionId) {
+    if (empty($this->rowData['contribution_owner_org_id'])) {
+      return;
+    }
+
+    $sqlQuery = "INSERT INTO `civicrm_value_multicompanyaccounting_ownerorg` (`entity_id` , `owner_organization`)
```

Comment:
What would happen if somebody adds  `contribution_owner_org_id` data but not enable multicompany accounting?

Reply:
that row won't be imported and once the import is done a CSV will be generated telling which rows were not imported and why. 

But I am not going to add check for this extension for performance reasons.